lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -108,24 +121,25 @@ mkJormungandrLayer mgr baseUrl = JormungandrLayer
     , getBlock = \blockId -> ExceptT $ do
         run (cGetBlock blockId)  >>= \case
             Left (FailureResponse e) | responseStatusCode e == status404 ->
-              return . Left $ ErrGetBlockNotFound blockId
+              return . Right $ Nothing
             x -> do
                 let ctx = safeLink api (Proxy @GetBlock) blockId
-                left ErrGetBlockNetworkUnreachable <$> defaultHandler ctx x
+                Just <<$>> defaultHandler ctx x
     , getDescendantIds = \parentId count -> ExceptT $ do
         run (cGetBlockDescendantIds parentId (Just count))  >>= \case
             Left (FailureResponse e) | responseStatusCode e == status404 ->
-              return . Left $ ErrGetDescendantsParentNotFound parentId
+              return . Right $ Nothing
             x -> do
                 let ctx = safeLink
                         api
                         (Proxy @GetBlockDescendantIds)
                         parentId
                         (Just count)
-                left ErrGetDescendantsNetworkUnreachable <$> defaultHandler ctx x
+                Just <<$>> defaultHandler ctx x
 
     }
   where
+    f <<$>> a = fmap (fmap f) a
```

Comment:
This is slightly _cryptic_ ... Try to avoid "weird" operators that aren't common. Especially without any precedence clearly defined. (fmap fn <$> ...) is really fine.